src/main/kotlin/com/depromeet/piki/user/service/UserService.kt

```
@@ -81,6 +81,26 @@ class UserService(
+        val id = UUID.randomUUID()
+        val nickname = generateUniqueGuestNickname()
+        return try {
+            userRepository.save(
+                User(
+                    id = id,
+                    nickname = nickname,
+                    profileImage = profileImage ?: dicebearUrl(id),
+                    identityType = IdentityType.MEMBER,
+                ),
+            )
+        } catch (e: DataIntegrityViolationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)